turbopack/crates/turbopack-node/src/transforms/postcss.rs

```diff
@@ -225,13 +228,70 @@ async fn extra_configs_changed(
     Ok(Vc::<Completions>::cell(configs).completed())
 }
 
+#[turbo_tasks::value]
+pub struct JsonKeySource {
```

Comment:
@sokra does this make sense or is it overengineered to create a new source for this?